nimdoc/tester.nim

```diff
@@ -75,7 +77,6 @@ let
                                      "$1/$2.nim" % [test2Dir, test2PrjName]],
                               buildIndex: @["--out:$1/$2/theindex.html" % [test2Dir, test2DocsDir],
                                             "$1/$2" % [test2Dir, test2DocsDir]])
-testNimDoc(test2Dir, test2DocsDir, test2Switches, defined(fixup))
+testNimDoc(test2Dir, test2DocsDir, test2Switches, fixup)
 
-# Check for failures
-if failures > 0: quit($failures & " failures occurred.")
+doAssert failures == 0, "$# failures occurred; see note in tester.nim regarding -d:nimTestsNimdocFixup" %  $failures
```

Comment:
Keep the `quit` version please, I really don't like to blur the line between bugs and error reporting. Error reports should not spit out stack traces. I mistype "testament cat typo" and I get a stack trace, it's really bad.

Reply:
done, but maybe in future i can do something similar to/generalize https://github.com/nim-lang/Nim/pull/17739 to give user a choice (not knowing where/how something quit is also bad for users); `quitOrRaise` controlled via a flag should hopefully address all such concerns